README.md

```diff
@@ -1,46 +1,57 @@
-# Getting Started with Create React App
+# Honda Tracker - booking app
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+The app for booking a car.
 
-## Available Scripts
+## Table of contents
 
-In the project directory, you can run:
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Setup](#setup)
+* [Illustrations](#illustrations)
 
-### `npm start`
+## General info
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+A simple app to track the time when users book Honda car. This application can be useful if several people use the same car.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+It is a responsive design app (mobile and desktop version).
+After logged-in user see a calendar with booking information. User can create, edit, delete a booking, complete a ride, see a booking details, choose settings. All booking information comes from Mongo DB.
 
-### `npm test`
+## Technologies
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+* React
+* Typescript
+* CSS: Tailwind
+* Forms: Formik
+* Intl.DateTimeFormat
+* RTK query
+* Postman client for backend requests
```

Comment:
Would be nice to export a Postman collection and store the resulting json file in the app

Reply:
changed